composer.json

```diff
@@ -2,7 +2,7 @@
     "name": "monerointegrations/moneropayment",
     "description": "Accept Monero",
     "require": {
-        "php": "~5.5.0|~5.6.0|~7.0.0"
+        "php": "~8.1.0||~8.2.0||~8.3.0"
```

Comment:
Is there a reason not to support php 7?

Reply:
The bounty says: "ensure compatibility with the latest magento version" and it seems they are already on 8.. https://github.com/magento/magento2/blob/a12063bdf7b965c9084596d241ccde7f19061093/app/code/Magento/Ui/composer.json#L8 (but other than not really I think.. but I did not test with 7.....)

Reply:
Seems fair